lib/src/executables/intl_message_migration.dart

```diff
@@ -78,6 +79,13 @@ void main(List<String> args) async {
       negatable: false,
       help: 'Forces ansi color highlighting of stderr. Useful for debugging.',
     )
+    ..addFlag(
+      _migrateConstants,
+      negatable: true,
+      defaultsTo: false,
```

Comment:
I must ask .. should we make this default to true?

Reply:
I was going to hide it behind a flag until we had that discussion.

Reply:
lovely. let's leave it behind a flag for now but I think we'll probably want to move it to default if the results we get from this addition are generally good.